src/urbit/app/chess.hoon

```diff
@@ -653,9 +674,22 @@
         ?~  archived-game  ~
         ``[%chess-game !>(u.archived-game)]
       ``[%chess-game !>(game.u.active-game)]
-    ::
-    ::  .^(arch %gy /=chess=/game)
-    ::  collect all the game-id keys
+  ::
+  ::  .^(noun %gx /=chess=/challenges/outgoing/noun)
+  ::  list challenges sent
+    [%x %challenges %outgoing ~]
+      ``[%chess-outgoing !>(~(tap by challenges-sent))]
+  ::
+  ::  .^(noun %gx /=chess=/challenges/incoming/noun)
+  ::  list challenges received
+    [%x %challenges %incoming ~]
+      ``[%chess-incoming !>(~(tap by challenges-received))]
```

Comment:
This shouldn't need two separates `mark`s. There should just be one `mark` called `%chess-challenges` which covers both cases, since the types are identical. Should suffice to delete one of `incoming.hoon` or `outgoing.hoon` and rename the remaining one to `challenges.hoon`.